example/ios/Runner/AppDelegate.swift

```diff
@@ -8,6 +9,13 @@ internal class AppDelegate: FlutterAppDelegate {
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
+        // configure for background audio
+        do {
+            try AVAudioSession.sharedInstance().setCategory(.playback)
+        } catch {
+            print("Setting category to AVAudioSessionCategoryPlayback failed.")
+        }
+
```

Comment:
This is a very similar version of configuring background playback on iOS. There will be a separate PR where this setup will be moved to the flutter part of the example.